c2corg_ui/static/js/addassociation.js

```diff
@@ -83,11 +83,12 @@ app.AddAssociationController = function(appApi, appDocument) {
  */
 app.AddAssociationController.prototype.associate = function(doc) {
   var parentId, childId;
+  var type = this.parentDoctype;
```

Comment:
``type`` is a bit confusing compared to ``doc['type']``. It's better to keep "parent" somewhere. At least ``parentType``.